HW_5/task_2.py

```diff
@@ -0,0 +1,21 @@
+"""
+Задача 2.
+Напишите однострочный генератор словаря, который принимает
+на вход три списка одинаковой длины: имена str, ставка int,
+премия str с указанием процентов вида «10.25%». В результате
+получаем словарь с именем в качестве ключа и суммой
+премии в качестве значения. Сумма рассчитывается
+как ставка умноженная на процент премии.
+"""
+
+
+def bonus_calc(names, salaries, rates):
+    yield {name: (salary / 100) * float(str(rate).strip("%")) for name, salary, rate in zip(names, salaries, rates)}
```

Comment:
Вы так создаете генератор словарей, зачем при этом возвращать еще генератор?
```suggestion
    return {name: (salary / 100) * float(str(rate).strip("%")) for name, salary, rate in zip(names, salaries, rates)}
```